src/index.js

```diff
@@ -0,0 +1,24 @@
+import { join, resolve } from 'path';
+
+import { fileListFromPath } from 'filelist-utils';
+
+const path = resolve(join(__dirname, '../data/'));
```

Comment:
Not sure why you was using resolve

Reply:
resolve is useless if the path is absolute (and it's guaranteed to be absolute when you use `__dirname` to construct it)